server/utils/validation/patientDataValidation/index.js

```diff
@@ -0,0 +1,26 @@
+const { parse, isDate } = require('date-fns');
+const { object, string, date } = require('yup');
+
+function parseDateString(value, originalValue) {
+  const parsedDate = isDate(originalValue)
+    ? originalValue
+    : parse(originalValue, 'yyyy-MM-dd', new Date());
+
+  return parsedDate;
+}
+
+const patientDataValidation = object({
+  patientId: string(),
+  firstName: string()
```

Comment:
FirstName - LastName -the phone should be required ,we make them as not null in database.